webgames/src/router/routes.ts

```diff
@@ -771,4 +772,12 @@ export const routes: RouteConfig[] = [
     tags: ["scroll", "reading", "stocks"],
     password: PASSWORD_StockMarketScroll,
   },
+  {
+    path: "/login-challenge",
+    title: "Verification Challenge",
+    description: "Can you decipher the verification code?",
+    icon: "🔐",
+    component: LoginChallenge,
+    tags: ["verification", "captcha"],
```

Comment:
Please add the `password` field here so that it will appear as an answer in the downloaded solutions.